README.md

```diff
@@ -120,21 +186,36 @@ The following table gives the a description of the fields and corresponding valu
+Reads were then aligned using STAR in two-pass mode to reference genome GRCh38. 
+While all MTP data is paired-end, methods are provided for single-end alignment if you are interested in processing your data in the same manner. 
+Transcripts are quantified using RSEM at both the gene and isoform level  with the GENCODE v27 annotation, except for the GTEx v8 samples which were not re-processed and are annotated using GENCODE v26. 
+Fusion calling is done using both Arriba and STAR-Fusion and then filtered for high confidence fusion calls using annoFuse. 
+QC metrics for the alignment are summarized using RNA-seQC. 
+If you would like to view the code in more detail, please see the GitHub release [OpenPBTA RNA-seq Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_RNAseq_workflow.cwl) and if you would like to run the pipeline, please see the [CAVATICA App](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/kfdrc-rnaseq-workflow/6). 
+Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation. 
 
 ### RNA Sequencing Data
 
 #### Fusions
 
-Gene fusions are called solely from RNA sequencing using the programs above. Fusions are filtered using custom R scripts. Fusion calls are retained if they are called by both STAR-Fusion and Arriba and if the fusion was specific and present in 3 or more samples in a single disease. Fusions were then annotated with gene and fusion specific information as well as whether they are known cancer genes from OncoKB, TCGA, and COSMIC. Summary frequencies are calculated using R. See [the fusion filtering documentation](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering) for specific code and further details. 
+Gene fusions are called either from NGS RNA sequencing using the methods described above or for the CHOP P30 panel data, using CHOP's Division of Genome Diagnostic's (DGD) Fusion Gene Panel. 
+No whole genome or whole exome NGS DNA sequencing is used to call fusions currently.
+We use annoFuse to filter out artifactual fusions, those with both gene partners having expression < 1 TPM, or fusions present in normal tissues. 
+We annotate genes as transcription factor, tumor suppressor, oncogene, and/or kinase and for kinase genes, identify whether or not the kinase domain is retained in the fusion.
+Fusions present in more than 4 broad histologies are also flagged and filtered out, as they may be non-specific.
+This final file of putative oncogenic fusions is the data utilized for the fusion frequency tables.
+Fusion panels were not reprocessed and were merged with our harmonized fusion calls.
+Summary frequencies are calculated using R. 
+See [the fusion filtering documentation](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering) for specific code and further details. 
```

Comment:
Does this updated paragraph address https://github.com/PediatricOpenTargets/documentation/pull/8#discussion_r1036587598 on fusion methods and https://github.com/PediatricOpenTargets/documentation/pull/8#discussion_r1036588027 on custom script language?

Reply:
great!